### PR TITLE
Declare the owners of the repository

### PR DESCRIPTION
**Pull request by maxgio92**
The owners file content declares the responsibilities of people that owns the code.
Approvers have the responsibility to integrate changes. Reviewers have the responsibility to reviewing changes.
More on this here: https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/#overview

### CODE REVIEW

**Comment by FedeDP** [comment]
And my axe!

**Comment by maxgio92** [comment]
Great @FedeDP! Thank you!

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvers:
+  - leogr
+  - maxgio92
```
Comment (FedeDP) [inline OWNERS:3]:
```suggestion
  - maxgio92
  - FedeDP
```

**Comment by maxgio92** [comment]
I guess @leogr we need your manual merge, as you and @FedeDP are still not approvers. WDYT?

Consequently and related, https://github.com/falcosecurity/test-infra/pull/1168.

**Comment by leogr** [comment]
> I guess @leogr we need your manual merge, as you and @FedeDP are still not approvers. WDYT?
> 
> Consequently and related, [falcosecurity/test-infra#1168](https://github.com/falcosecurity/test-infra/pull/1168).

Yes. I'm going to manually merge this since there is no other technical way to do so.

**Pull request merged**